py_irt/cli.py

```diff
@@ -122,7 +122,8 @@ def evaluate(
     seed: int = 42,
     train_size: float = 0.9,
 ):
-    console.log(f"model_type: {model_type} data_path: {data_path}")
+    # removed console.log since data_path is not passed into this function as an 
```

Comment:
This comment doesn't need to be here and the model type should still be logged.

Reply:
Resolved in next commit. Does everything else look good?